client/admin/channel-view.tsx

```diff
@@ -69,6 +70,15 @@ function getChannelMessages(
       type: '@users/loadUsers',
       payload: result.mentions.concat(result.users),
     })
+    dispatch({
```

Comment:
I'm 99% sure we've already been over this before, you shouldn't be using the main reducer to show messages to admins. By doing this you are polluting their normal app state. It is okay to do with users because these are data that does not differ depending on the user requesting it, and the data is small (so it's okay to keep around for a long time/forever). With chat, that's not the case, and adding this stuff to the reducer state means it will never be removed, it will just permanently leak memory any time an admin views a channel. I guess you should replace this with a note so that we stop trying to build this in every subsequent PR.

In general, NEVER use redux to store admin state. It is basically never needed to be shared between UIs, and is not necessary to cache, we don't need to use redux for it.

Reply:
I actually don't even understand why you did this? The component below wasn't even changed around any of this, so it's still doing the right thing (saving things to its local state)

Reply:
> I actually don't even understand why you did this?

I honestly have no idea. I have no memory of writing this code and it doesn't even look like something that would be produced by resolving the merge conflicts or something. Weird.